api/pandas/covariate_dataframe.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Covariates for Places as pd.DataFrame
+nav_order: 3
+parent: Pandas
+grand_parent: API
+---
+
+# Get Covariates for Places as pandas DataFrame
+
+## `datacommons_pandas.build_covariate_dataframe(dcids, stats_vars)`
```

Comment:
nit: we used places instead of dcids in the REST api

Reply:
Thanks for the catch